connectors/cassandra/cassandra-api/pom.xml

```diff
@@ -0,0 +1,33 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.teiid.connectors</groupId>
+        <artifactId>cassandra</artifactId>
+        <version>14.0.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>cassandra-api</artifactId>
+    <name>Cassandra API</name>
+    <description>The java API for the Cassandra</description>
+    <dependencies>
+        <dependency>
+            <groupId>org.teiid</groupId>
+            <artifactId>teiid-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.teiid</groupId>
+            <artifactId>teiid-common-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.spec.javax.resource</groupId>
+            <artifactId>jboss-connector-api_1.7_spec</artifactId>
+        </dependency>
```

Comment:
See if you can remove this dependency, as this is JEE specific jar and we do not to want bring that dependency. See what kind of errors you run into and fix